backend/api/Models/Parts/ParcelModel.cs

```diff
@@ -28,13 +30,13 @@ public override bool Equals(object obj)
         public bool Equals([AllowNull] ParcelModel other)
         {
             return other != null &&
-                   Id == other.Id &&
-                   PID == other.PID &&
-                   StatusId == other.StatusId &&
-                   ClassificationId == other.ClassificationId &&
-                   Latitude == other.Latitude &&
-                   Longitude == other.Longitude &&
-                   Description == other.Description;
+                Id == other.Id &&
+                PID == other.PID &&
+                StatusId == other.StatusId &&
+                ClassificationId == other.ClassificationId &&
+                Latitude == other.Latitude &&
+                Longitude == other.Longitude &&
+                Description == other.Description;
         }
```

Comment:
Question: shouldn't the new **PIN** property be included in the `Equals` method?

Reply:
Same for the previous file...

Reply:
probably yes, and why I'm not a fan of this implementation.

Reply:
fixed